common.gradle

```diff
@@ -81,35 +83,61 @@ if(!rootProject.hasProperty("useLocalDependencies")) {
 ext.gearvrfVersion='3.1.1-SNAPSHOT'
 project.ext.jomlVersion = "1.9.1-SNAPSHOT"
 
-dependencies {
-    compile "org.joml:joml-android:${jomlVersion}"
-    if(rootProject.useLocalDependencies) {
+if(rootProject.useLocalDependencies) {
+    dependencies {
+        compile "org.joml:joml-android:${jomlVersion}"
         compile(name: 'framework-debug', ext: 'aar')
-    } else {
-        compile "org.gearvrf:framework:$gearvrfVersion"
-    }
 
-    if (rootProject.hasProperty("only_daydream")) {
-        if(rootProject.useLocalDependencies) {
+        if (rootProject.hasProperty("only_daydream")) {
             compile(name: 'backend_daydream-debug', ext: 'aar')
+            compile 'com.google.vr:sdk-base:1.10.0'
+            compile 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-7'
         } else {
-            compile "org.gearvrf:backend_daydream:$gearvrfVersion"
-        }
-        compile 'com.google.vr:sdk-base:1.10.0'
-        compile 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-7'
-    } else {
-        if (rootProject.hasProperty("no_oculus")) {
-            compile(name: 'backend-debug', ext: 'aar')
-        } else { //default oculus+daydream
-            if(rootProject.useLocalDependencies) {
+            if (rootProject.hasProperty("no_oculus")) {
+                compile(name: 'backend-debug', ext: 'aar')
+            } else { //default oculus+daydream
                 compile(name: 'backend_daydream-debug', ext: 'aar')
                 compile(name: 'backend_oculus-debug', ext: 'aar')
-            } else {
-                compile "org.gearvrf:backend_daydream:$gearvrfVersion"
-                compile "org.gearvrf:backend_oculus:$gearvrfVersion"
+                compile 'com.google.vr:sdk-base:1.10.0'
+                compile 'com.google.protobuf.nano:protobuf-javanano:3.0.0-alpha-7'
             }
+        }
+    }
+} else if(rootProject.hasProperty("buildGvrfAndDemos")) {
```

Comment:
I still prefer the aars since they save a good chunk of time when rebuilding the demos. The automated framework build produces aar artifacts. The automated demos build is set up to use those aars. One day we can refactor the thing. The goal of this pull request is just to make sure debugging works.

If you'd like you could propose a change in a pull request but we need to take into account the various use cases we have for the aars.